lib/TypeNames/ModelTypeNames.cpp

```diff
@@ -30,6 +30,8 @@
 #include "revng/TypeNames/LLVMTypeNames.h"
 #include "revng/TypeNames/PTMLCTypeBuilder.h"
 
+Logger<> VariableNamingLog("variable-naming");
```

Comment:
Consider outlining all the users of this to the `.cpp` file and renaming this to `Log`.

Reply:
Done.